From: Andrew Morton <akpm@linux-foundation.org>
Subject: userfaultfd-retry-copying-with-locks-dropped-in-mfill_atomic_pte_copy-fix-fix
Date: Wed Mar 25 03:23:43 PM PDT 2026

update mfill_copy_folio_retry() for
userfaultfd-introduce-mfill_get_pmd-helper-fix.patch

Link: https://lkml.kernel.org/r/20260316173829.1126728-1-avagin@google.com
Signed-off-by: Mike Rapoport <rppt@kernel.org>
Cc: Andrei Vagin <avagin@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/userfaultfd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/userfaultfd.c~userfaultfd-retry-copying-with-locks-dropped-in-mfill_atomic_pte_copy-fix-fix
+++ a/mm/userfaultfd.c
@@ -427,7 +427,7 @@ static int mfill_copy_folio_retry(struct
 	if (err)
 		return err;
 
-	err = mfill_get_pmd(state);
+	err = mfill_establish_pmd(state);
 	if (err)
 		return err;
 
_
